keyboards/iris/rules.mk

```
@@ -61,10 +32,15 @@ MIDI_ENABLE = no            # MIDI controls
 AUDIO_ENABLE = no           # Audio output on port C6
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight. 
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
 
 SPLIT_KEYBOARD = yes
 
 DEFAULT_FOLDER = iris/rev2
+
+# VIA Support - Comment these 3 lines out to disable
+RAW_ENABLE = yes
+DYNAMIC_KEYMAP_ENABLE = yes
+SRC += keyboards/wilba_tech/wt_main.c
```

Well, I think this would be best: 

```suggestion
ifeq ($(strip $(DYNAMIC_KEYMAP_ENABLE)), yes)
  WILBA_TECH_ENABLE = yes
endif
ifeq ($(strip $(RAW_ENABLE)), yes)
  WILBA_TECH_ENABLE = yes
endif
ifeq ($(strip $(WILBA_TECH_ENABLE)), yes)
    SRC += keyboards/wilba_tech/wt_main.c
endif
```

It's more complicated, but it would only include the source file, if either of these options are enabled.  If not, then it wouldn't be added. 

I would prefer this.... in a partially selfish reason.

---

It doesn't work in <keyboards>/rules.mk though, because these conditionals get handled before user rules.mk.

---

you're correct :(